### ISSUE TITLE
Move Item Here does not work (Option+Command+V)

### ISSUE THREAD

**Issue by nitrostudios** [doublecmd/doublecmd#1480]
### Version

Double Commander
Version: 1.2.0 alpha
Revision: 11223
Commit: a5009f63a
Build date: 2024/02/13
Lazarus: 3.99.0.0
FPC: 3.2.2
Platform: aarch64-Darwin-cocoa
OS version: Mac OS X 12.6.6

### Description

When using shortcuts to copy (Command+C) and Move Item Here (Option+Command+V) it does not work on macOS. Is there a way to set some kind of shortcut, please? Thank you.

**Comment by rich2014** [comment]
@nitrostudios 

now `Command+X` for `Cut`, and `Command+V` for `Move`

**Comment by rich2014** [comment]
@alexx2000 

is there a Command similar to `Move from Clipboard to Here`?

**Comment by alexx2000** [comment]
No, of course, `Command+V` pastes previously copied/cut out item, action determined by first command: `Command+C` or `Command+X`.

**Comment by rich2014** [comment]
@alexx2000 

in Finder:
`Command+C` for `Copy to Clipboard`
`Command+V` for `Copy from Clipboard`
`Option+Command+V` for `Cut from Clipboard`

should we consider adding support for this command `Cut from Clipboard`?

**Comment by alexx2000** [comment]
I don't sure, it conflicts with DC logic.

**Comment by rich2014** [comment]
it's the difference in usage habits between windows and macOS.

the addition of the `Cut from Clipboard` command seems harmless. if the user does not have this habit, the user will be unaware.
